lib/matplotlib/rcsetup.py

```diff
@@ -897,7 +897,7 @@ def validate_hist_bins(s):
     'lines.solid_capstyle':  ['projecting', validate_capstyle],
     'lines.dashed_pattern':  [[2.8, 1.2], validate_nseq_float()],
     'lines.dashdot_pattern': [[4.8, 1.2, 0.8, 1.2], validate_nseq_float()],
-    'lines.dotted_pattern':  [[1.2, 0.6], validate_nseq_float()],
+    'lines.dotted_pattern':  [[1.1, 1.1], validate_nseq_float()],
```

Comment:
In the line above, I expect the second half of the dashdot pattern should match the dot pattern, or if anything, should be even a little more stretched out along the line.  The 0.8 can go to 1.2.